2018-10-25-Ticha-Review

```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.  
+*Ticha: a digital text explorer for Colonial Zapotec*.  
+First edition.(http://ticha.haverford.edu/)
+
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Thicha is a digital platform  
+that was and is being created by an interdisciplinary team of scholars from different fields in the United States  
+and Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+Ticha analyzes data through two main technological platforms: FLEx which is Fieldworks Language Explorer, a system for  
+lexical and grammatical analysis, with Text Encoding Initiative standards for paleographic and translational representations  
+of texts as stated in About the Project. The corpus presented is from the colonial period which focus on three main works,  
+all of these texts come from the geographic region of Central Valley Zapotec Oaxaca. The first one is [Arte], a book published  
+in 1578 by Juan de Cordova, a Spanish priest in the Dominican order. The second is Doctrina christiana en lengua castellana y  
+çapoteca by Dominican priest Pedro de Feria. The third are the handwritten texts mainly written by native Zapotec speakers in  
+Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital platform is organized  
+in the following three main categories: About, Colonial Zapotec, Explore the texts and Bibliography. The about presents  
+information about the project, the team, acknowledgments and a contact us. In addition to this information, at the bottom of  
+the about page, there are videos about the project with the team.
```

Comment:
Think about reorganizing the information in this paragraph in terms of priority. I see three main descriptive points: the topic of the collection, the content of the collection, and the encoding system. I would organize them so the topic comes first, followed by content, and finally encoding.